Observer/Config/Save.php

```diff
@@ -98,40 +107,44 @@ public function execute(Observer $observer)
     public function doYotpoApiKeyValidation(Observer $observer)
     {
         $changedPaths = (array)$observer->getEvent()->getChangedPaths();
-        if ($changedPaths && $this->isYotpoKeysChanged($changedPaths)) {
-            $this->cacheTypeList->cleanType(Config::TYPE_IDENTIFIER);
-            $this->appConfig->reinit();
+        if ($changedPaths) {
             $scopeDetails = $this->getScopes($observer);
             $scopeId = (int) $scopeDetails['scope_id'];
```

Comment:
Please move the `(int)` casting to `Config::Main#getScopes`
```
public function getScopes(Observer $observer): array
{
...
$return['scope_id'] = (int)  $scopeId;
...
}
```

```suggestion
            $scopeId = $scopeDetails['scope_id'];
```

Reply:
This is done